From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Fri, 13 Dec 2024 15:52:31 +0100
Subject: [PATCH] HACK: dependencies: trust pkgconfig when looking for zlib

If it's filtered in pkg-config, don't try to find it in another way.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 mesonbuild/dependencies/dev.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mesonbuild/dependencies/dev.py b/mesonbuild/dependencies/dev.py
index 4d219a58967f..85f6e2502941 100644
--- a/mesonbuild/dependencies/dev.py
+++ b/mesonbuild/dependencies/dev.py
@@ -818,7 +818,7 @@ packages['gmock'] = gmock_factory = DependencyFactory(
 
 packages['zlib'] = zlib_factory = DependencyFactory(
     'zlib',
-    [DependencyMethods.PKGCONFIG, DependencyMethods.CMAKE, DependencyMethods.SYSTEM],
+    [DependencyMethods.PKGCONFIG],
     cmake_name='ZLIB',
     system_class=ZlibSystemDependency,
 )
